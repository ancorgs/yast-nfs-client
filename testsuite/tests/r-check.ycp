/**
 * Module:
 *   NFS client configuration
 *
 * Summary:
 *   Routines testuite
 *
 * Authors:
 *   Martin Vidner <mvidner@suse.cz>
 *
 * $Id$
 */
{
    include "testsuite.ycp";
    include "nfs/routines.ycp";

    DUMP ("check_options");
    TEST (``(check_options ("")), [], nil);
    TEST (``(check_options ("defaults")), [], nil);
    TEST (``(check_options ("nolock,bg")), [], nil);
    TEST (``(check_options ("nolock,nobg")), [], nil);
    TEST (``(check_options ("nolock,rsize=8192")), [], nil);
    TEST (``(check_options ("nolock, bg")), [], nil);
    TEST (``(check_options ("nolock,unknownoption")), [], nil);
    TEST (``(check_options ("nolock,unknownassignment=true")), [], nil);
    TEST (``(check_options ("nolock,rsize=")), [], nil);
    TEST (``(check_options ("nolock,two=equal=signs")), [], nil);
}
