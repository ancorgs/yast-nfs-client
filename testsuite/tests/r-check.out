Dump	check_options
Return	Empty option strings are not allowed.
Return	
Return	
Return	
Return	
Return	Unknown option:  bg
Return	Unknown option: unknownoption
Return	Unknown option: unknownassignment
Return	Empty value for option: rsize
Return	Unknown option: two
Dump	CheckHostName
Return	true
Log	The hostname entered is invalid. It must be
Return	false
Log	The hostname entered is invalid. It must be
Return	false
Log	The hostname entered is invalid. It must be
Return	false
Return	true
Log	The hostname entered is invalid. It must be
Return	false
Return	true
Log	The hostname entered is invalid. It must be
Return	false
Return	true
Log	The hostname entered is invalid. It must be
Return	false
Return	true
Return	true
Log	The hostname entered is invalid. It must be
Return	false
Dump	FormatHostnameForFstab
Return	[::1]
Return	[::1]
Return	127.0.0.1
Return	suse.de
